spark/src/test/java/org/apache/iceberg/actions/TestRemoveOrphanFilesAction.java

```diff
@@ -528,6 +566,71 @@ public void testRemoveOrphanFilesWithRelativeFilePath() throws IOException, Inte
     Assert.assertTrue("Invalid file should be present", fs.exists(new Path(invalidFiles.get(0))));
   }
 
+  @Test
+  public void testRemoveOrphanFilesWithSpecialCharsFilePath() throws IOException, InterruptedException {
+    File whiteSpaceDir = null;
+    try {
+      whiteSpaceDir = new File(tableDir.getAbsolutePath() + "/white space");
```

Comment:
Not sure we need to be careful with cleanup here, couldn't we just use temp.newDir and then skip the cleanup?

Reply:
sure, used tmp now.